src/mantine-core/src/components/Tabs/TabControl/TabControl.styles.ts

```diff
@@ -28,33 +29,35 @@ export default createMemoStyles({
     padding: [0, theme.spacing.md],
     fontSize: theme.fontSizes.sm,
     cursor: 'pointer',
+    width: orientation === 'vertical' && '100%',
```

Comment:
Will break jss styles in some cases (cannot remember these case but pretty sure that we cannot use this syntax), change this instead to 

```tsx
width: orientation === 'vertical' ?'100%' : undefined,
```

Reply:
Note: I prefer the ternary syntax over the short-circuit (`&&`) syntax, but most of what I saw was short-circuited. I'll change these up.